cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -130,8 +130,9 @@
 default['travis_build_environment']['bats_git_repository'] = \
   'https://github.com/sstephenson/bats.git'
 
-default['travis_build_environment']['hhvm_enabled'] = false
-default['travis_build_environment']['hhvm_package_name'] = 'hhvm'
+default['travis_build_environment']['hhvm']['enabled'] = false
+default['travis_build_environment']['hhvm']['package_name'] = 'hhvm'
+default['travis_build_environment']['hhvm']['keep_repo'] = false
```

Comment:
fwiw I try to avoid nesting chef attributes like this, as the automatic hash initialization comes with gotchas